### ISSUE TITLE
Setting locale store in preload method causes 'window is not defined' error

### ISSUE THREAD

**Issue by rodoch** [kaisermann/svelte-i18n#55]
Since upgrading to svelte-i18n v3 one or two of my Sapper apps appear to have experienced a regression. When the locale value is set in the preload method (in the interests of SSR) the program throws a `window is not defined` error. Previously this did not occur under the same circumstances.

```html
<script context='module'>
  import { locale, waitLocale } from 'svelte-i18n'

  export async function preload(page, session) {
    locale.set('ga'); // intention being to set value from session variable
    waitLocale();
  }
</script>
```

The error:

```
ReferenceError: window is not defined
    at Object.x.set (D:\PATH\node_modules\svelte-i18n\dist\runtime.cjs.js:16:1820)
    at Object.preload (D:\PATH\__sapper__\dev\server\server.js:5821:20)
    at handle_page (D:\PATH\__sapper__\dev\server\server.js:12096:29)
    at handle_error (D:\PATH\__sapper__\dev\server\server.js:11984:3)
    at handle_page (D:\PATH\__sapper__\dev\server\server.js:12148:5)
```

Occuring on: all browsers, Mac OS and Windows, v3.1 and 3.2 (I haven't tested 3.0), Rollup build.

I have created a minimal repro here:
https://github.com/rodoch/svelte-i18n-bug-repro

I have managed to fix the issue by adding a window is undefined check to line 69 of  `/svelte-i18n/src/runtime/stores/locale.ts`:

```js
    if (typeof window !== 'undefined' && getCurrentLocale() != null && loadingDelay) {
      loadingTimer = window.setTimeout(() => $isLoading.set(true), loadingDelay)
    } else {
      $isLoading.set(true)
    }
```

Happy to open a PR if you think this is actually the solution and not a hack!

**Comment by kaisermann** [comment]
Sorry for taking so long to get to this. Fix released in `3.0.3` 🎉

**Comment by rodoch** [comment]
Great stuff, thanks!
